app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController 
+    def show
+        @user = User.find(params[:id])
+    end
+
+    def create
+        @user = User.create! user_params.require(:user)
```

Comment:
- Sử dụng user_params
- Không dùng create! ngay vì có thể vướng validate nên lưu bị lỗi, nên dùng User.new hoặc .build, sau đó dùng save
Ex:
if @user.save
 redirect_to xxx
else
 flash[:error] = "Fail to create user"
end